README.md

```diff
@@ -51,24 +53,25 @@ The app should now be running on port 8080.
 <a id="markdown-configuration" name="configuration"></a>
 ## Configuration
 
-Images of this project are built, and hosted on [DockerHub](https://cloud.docker.com/u/asecurityteam/repository/docker/asecurityteam/awsconfig-filterd).
-
-This code functions as a stand-alone Lambda function, and can be deployed to AWS Lambda directly.
-To run in the AWS Lambda environment, create a new Go project, import this project as a dependency, and run the lambda
-using the aws-lambda-sdk:
-
-```go
-func main() {
-    resourceFiltererComponent := &filter.ResourceTypeFiltererComponent{}
-    resourceTypeFilterer := new(filter.ResourceTypeFilterer)
-    _ = settings.NewComponent(ctx, source, resourceFiltererComponent, resourceTypeFilterer)
-    configFilterer := &v1.ConfigFilterHandler{
-        LogFn:          <LOGGER_PROVIDER>,
-        StatFn:         <STATS_PROVIDER>,
-        ConfigFilterer: resourceTypeFilterer,
-    }
-  lambda.Start(configFilterer.Handle)
-}
+Images of this project are built, and hosted on [DockerHub](https://cloud.docker.com/u/asecurityteam/repository/docker/asecurityteam/awsconfig-filterd). The system is configured using environment variables. The following are all of the configuration options for the system:
+
+```bash
+# (bool) Use the Lambda SDK to start the system.
+AWSCONFIGFILTERD_LAMBDAMODE="false"
+# ([]string) List of enabled filters.
+AWSCONFIGFILTERD_FILTER_ENABLED="resourcetype"
```

Comment:
Question: How would you go about configuring an `AnyMatch` filter which takes a `ResourceType` filter in this context?

Reply:
Currently, you don't. This is set up such that additional filters are all combined using the `AnyMatch` underneath. The way I understood how this system was working is that it discards all events by default. A filter in this system is something that allows a message to pass through to the next stage. Because of this, I defaulted it to `AnyMatch` for whichever filters are enabled.

If there's a case for complex structures like `AnyMatch` and `AllMatch` being chained to form boolean expressions then we'd need to re-evaluate the loading strategy for settings. I don't believe the high-level component form can handle this today and we'd likely need to drop down in the the lower level APIs to make that happen.

Reply:
This makes perfect sense. I missed that the `AnyMatch` filter was being used in `FilterComponent.New`, and thus mistakenly believed it was unused.